### ISSUE TITLE
Windows buildbot failing with pygments

### ISSUE THREAD

**Issue by psavery** [OpenChemistry/tomviz-superbuild#299]
Unfortunately, the windows buildbot is failing with the newly added pygments, and it is failing [without giving much information](https://open.cdash.org/viewBuildError.php?buildid=6198998).

It looks like it is indeed downloading the pygments source and unzipping it.

Running `ninja` manually on the buildbot shows this:
```
FAILED: pygments/src/pygments-stamp/pygments-build
cmd.exe /C "cd /D C:\bbd\fff8b192\build\pygments\src\pygments && "C:/dashboards/support/cmake-3.13.2-win64-x64/bin/ctest.exe" --launch --target-name TARGET_NAME --build-dir C:\bbd\fff8b192\build --output C:\bbd\fff8b192\build\pygments\src\pygments-stamp\pygments-build -- env CPPFLAGS=-IC:/bbd/fff8b192/build/install/include LDFLAGS=-LC:/bbd/fff8b192/build/install/lib setup.py build && "C:/dashboards/support/cmake-3.13.2-win64-x64/bin/ctest.exe" --launch --target-name TARGET_NAME --build-dir C:\bbd\fff8b192\build --output C:\bbd\fff8b192\build\pygments\src\pygments-stamp\pygments-build -- C:\dashboards\support\cmake-3.13.2-win64-x64\bin\cmake.exe -E touch C:/bbd/fff8b192/build/pygments/src/pygments-stamp/pygments-build"
```

It appears that the second part (delimited by `&&`) of the command is failing, in other words:
```
"C:/dashboards/support/cmake-3.13.2-win64-x64/bin/ctest.exe" --launch --target-name TARGET_NAME --build-dir C:\bbd\fff8b192\build --output C:\bbd\fff8b192\build\pygments\src\pygments-stamp\pygments-build -- env CPPFLAGS=-IC:/bbd/fff8b192/build/install/include LDFLAGS=-LC:/bbd/fff8b192/build/install/lib setup.py build
```

However, there is no output indicating the cause of the failure. If I try to build it manually with `python.exe setup.py build` using the superbuild's python, it seems to build okay.

If I add `-VV` to that failing ctest command, it outputs that no tests were found.

In short: I'm not sure exactly what's going on.

**Comment by psavery** [comment]
So, I think I found that the `pv_python_executable` cmake variable doesn't actually get set on Windows, since it has its own `python.cmake` file. I guess we have never ran `python setup.py build` on Windows before. I did not realize this...

I think I can fix it by making sure that cmake variable is defined, so that the python executable appears correctly.

**Comment by psavery** [comment]
Even if the python executable does appear, it still fails with no error messages when the two "env" arguments "CPPFLAGS=" and "LDFLAGS=" are present. If they are removed, the build runs and succeeds.

**Comment by psavery** [comment]
Yeah, it's because there's no `env` command in windows... only in bash...

**Comment by psavery** [comment]
This issue is fixed with #300. It is confirmed that pygments is now working on Windows!
